README.md

```diff
@@ -84,114 +53,100 @@ To remove a dependency from your project, you can use the `poetry remove` comman
-> the related documentation.
+If this plugin is not installed, you may have to run poe as a script within the Poetry environment using `poetry run poe <command>`.
 
 ## Development
 
-This sections describes how to develop and run the kyma-companion fastapi app.
-
-### Redis Server
+### Redis Server 
 
-Before running the application, the Redis server must be provided that is used to store the conversation with a LLM.
-Therefore, REDIS_URL has to be provided as an environment variable.
+Before running the application, the Redis server must be provided. It is used to store the conversation with a large language model (LLM).
+Therefore, **REDIS_URL** must be provided as an environment variable.
 
-Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) how to create a Redis server. For example,
-`REDIS_URL="redis://{host or ip}:6379"`
+Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) <!--the link must be replaced when the OS documentation is available --> on how to create a Redis server. 
+For example, `REDIS_URL="redis://{host or ip}:6379"`
 
-### Running
+### Running Kyma Companion Locally
 
-You can start the FastApi application with the following options:
+You can start the FastAPI application with the following options:
 
 ```bash
 poetry run fastapi dev src/main.py --port 8000
 ```
 
-or, directly with FastApi:
-
-```bash
-fastapi dev src/main.py --port 8000
-```
-
-or, with a poe task:
+Or, with a poe task:
 
 ```bash
 $ poetry poe run-local
 ```
 
-Prefer running it with poetry as it activates and uses its virtual environment if not activated yet.  
-**NOTE:** it cannot be run with python directly.
+It is recommended to run Kyma Companion with Poetry as it activates and uses its virtual environment if not activated yet.
+
+> [!NOTE] You cannot run it with Python directly.
 
 ### Debugging
 
-The FastAPI application can be debugged with PyCharm or VS Code. Follow the following guidelines for these IDEs:
+As the companion uses the FastAPI framework, refer to the following documentation on how to debug the application with the respective IDE:
 
-- PyCharm: https://www.jetbrains.com/help/pycharm/fastapi-project.html#create-project
-- VS Code: https://code.visualstudio.com/docs/python/tutorial-fastapi
+* [PyCharm](https://www.jetbrains.com/help/pycharm/fastapi-project.html#create-project)
+* [VS Code](https://code.visualstudio.com/docs/python/tutorial-fastapi)
 
 ### Configuration
 
-For local development LLM models can be configured inside the config/models.json file.  
-**NOTE:** Don't use it to configure models in dev, stage or prod clusters.
+For local development, LLM models can be configured inside the `config/models.json` file.  
+> [!NOTE] Don't use the <code>config/models.json</code> file to configure models in dev, stage, or prod clusters.
```

Comment:
Why? 
If it's dangerous to do that, consider using a "Caution" and explain why it's a bad idea.

Reply:
@grischperl Can you help with this question as well?

Reply:
The setup for dev, stage and prod will be different. Maybe this can be changed to simply say: "This file is for local development only." without mentioning dev, stage and prod?